packages/lambda-powertools-pattern-basic/supplement-csv.js

```diff
@@ -0,0 +1,23 @@
+const csvToKeyValuesArray = csv =>
+  csv.split(',').filter(x => x).map(pair => pair.split(':'))
+
+const keyValuesArrayToCsv = pairs =>
+  pairs.map(x => x.join(':')).join(',')
+
+const combineUniqueKeyValueArray = (original, additional) =>
```

Comment:
`existing` instead of `original` so we name things consistently

Reply:
Also, when we call this function on ln17, we pass in `additionalNormalised` first, which we call `original` here. Is that not the wrong way round?